docs/reference/types/subscription.md

```diff
@@ -55,6 +55,7 @@ nav_order: 3
 | `transport` | The transport plugin responsible for event delivery (WebSockets, Webhooks, JMS, NATS etc.) | `string` |
 | `filter` | Server-side filter to apply to events | [`SubscriptionFilter`](#subscriptionfilter) |
 | `options` | Subscription options | [`SubscriptionOptions`](#subscriptionoptions) |
+| `tlsConfigName` | The name of an existing TLS configuration associated to the namespace to use when performing the request in the case of webhooks | `string` |
```

Comment:
I think this is great 👍

Reply:
I believe this should be moved down to the Webhook specific config though, rather than at this level:
https://hyperledger.github.io/firefly/v1.2.0/reference/types/subscription.html#webhookinputoptions

`WebhookInputOptions` is the object

Reply:
Sure - will do that